src/app/api/composio/connections/route.ts

```
@@ -0,0 +1,72 @@
+      // We might want to filter by active status if the API supports it,
+      // or filter manually after fetching.
+      // active: true, // Example: if supported by the SDK call
+    });
+
+    // The actual response structure from `list` needs to be verified.
+    // Assuming it's an array of connection objects.
+    // Let's call the items `items` if it's a paginated-like response, or it could be a direct array.
+    // For now, let's assume `connectionsResponse` is the array or has an `items` property.
+    const connections = Array.isArray(connectionsResponse)
+      ? connectionsResponse
+      : (connectionsResponse as any)?.items || [];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)